test/tests/providers/capi/capz/platform/atlas/alerting-rules/logging-pipeline.rules.test.yml

```diff
@@ -0,0 +1,121 @@
+---
```

Comment:
Same file as `test/tests/providers/global/platform/atlas/alerting-rules/logging-pipeline.rules.test.yml` just added the `dashboardQueryParams: orgId=2` annotation